roles/confluent.variables_handlers/defaults/main.yml

```diff
@@ -106,6 +106,13 @@ kafka_broker:
   log4j_file: /etc/kafka/kafka_server_log4j.properties
   jaas_file: /etc/kafka/kafka_server_jaas.conf
 kafka_broker_schema_validation_enabled: false
+kafka_broker_metrics:
+  enabled: true
+  listener: 
+    bootstrap_servers: "{% for host in groups['kafka_broker'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{kafka_broker_listeners[kafka_connect_kafka_listener_name]['port']}}{% endfor %}"
```

Comment:
I love this! Didn't know this level of jinja templating can be moved into the variables files. We should be moving everything into variables which will allow so much customization out of the box. Just realized how helpful this could be